test/Artifacts.jl

```diff
@@ -49,3 +49,15 @@ function water_conservation_test_data_path(; context = nothing)
     dirichlet_datapath = joinpath(folder_path, "ref_soln_dirichlet.csv")
     return flux_datapath, dirichlet_datapath
 end
+
+
+"""
+    pmodel_unittests_path(; context = nothing)
+
+Returns the filepaths for input-output pairs for the P-model. 
```

Comment:
Could you add a sentence in this docstring explaining that the data in the artifact comes from the R implementation of the P model?